### PR TITLE
tests(ticdc): fix table filter in tests

### LINKED ISSUE #8272: some filtering rules for sync_diff in the integration test are incorrect
### What did you do?

run consistent_replicate_storage_file

### What did you expect to see?

Check all tables in tests
```log
The table structure and data in following tables are equivalent

+--------------------------------------------------+---------+-----------+
|                      TABLE                       | UPCOUNT | DOWNCOUNT |
+--------------------------------------------------+---------+-----------+
| `consistent_replicate_storage_file`.`t1`         |      15 |        15 |
| `consistent_replicate_storage_file`.`t2`         |       8 |         8 |
| `consistent_replicate_storage_file`.`usertable2` |    5000 |      5000 |
| `consistent_replicate_storage_file`.`usertable`  |    5000 |      5000 |
+--------------------------------------------------+---------+-----------+
```

### What did you see instead?

```log
The table structure and data in following tables are equivalent

+--------------------------------------------------+---------+-----------+
|                      TABLE                       | UPCOUNT | DOWNCOUNT |
+--------------------------------------------------+---------+-----------+
| `consistent_replicate_storage_file`.`t1`         |      15 |        15 |
| `consistent_replicate_storage_file`.`t2`         |       8 |         8 |
| `consistent_replicate_storage_file`.`usertable`  |    5000 |      5000 |
+--------------------------------------------------+---------+-----------+
```

### Versions of the cluster

Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

```console
(paste TiDB cluster version here)
```

Upstream TiKV version (execute `tikv-server --version`):

```console
(paste TiKV version here)
```

TiCDC version (execute `cdc version`):

```console
(paste TiCDC version here)
```

### PR DESCRIPTION
**Pull request by CharlesCheung96**
### What problem does this PR solve?

Issue Number: close #8272 

### What is changed and how it works?
- replace `USERTABLE` with `usertable`: sync_diff_inspector's table filter rules are case-sensitive, so we need to ensure that all table names in tests are the same as those in the diff configuration.

#### Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
 `None`.
```

### CODE REVIEW

File: tests/integration_tests/big_txn/run.sh [hunk 1]
```diff
@@ -36,11 +36,11 @@ function run() {
 		run_kafka_consumer $WORK_DIR "kafka://127.0.0.1:9092/$TOPIC_NAME?protocol=open-protocol&partition-num=4&version=${KAFKA_VERSION}&max-message-bytes=10485760"
 	fi
 
-	check_table_exists "big_txn.USERTABLE" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
-	run_sql "CREATE TABLE big_txn.USERTABLE1 LIKE big_txn.USERTABLE" ${UP_TIDB_HOST} ${UP_TIDB_PORT}
-	run_sql "INSERT INTO big_txn.USERTABLE1 SELECT * FROM big_txn.USERTABLE" ${UP_TIDB_HOST} ${UP_TIDB_PORT}
+	check_table_exists "big_txn.usertable" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT}
```
Comment (amyangfei) [inline tests/integration_tests/big_txn/run.sh:39]: Why changing it from upper case to lower case, TiDB table name is case insensitive by default.

Comment (CharlesCheung96) [reply tests/integration_tests/big_txn/run.sh:39]: sync_diff_inspector's table filter rules are case-sensitive, so we need to ensure that all table names in tests are the same as those in the diff configuration.

**Review by asddongmen — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

**Pull request merged**